File: official/docs/csharp/v5/pagination/get-next-page.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            // Get first page of results
            EasyPost.Parameters.Shipment.All parameters = new()
            {
                PageSize = 5
            };

            EasyPost.Models.API.ShipmentCollection shipmentCollection = await client.Shipment.All(parameters);

            // Provide the previous results page to move onto the next page
            ShipmentCollection nextPage = await client.Shipment.GetNextPage(shipmentCollection);

            Console.WriteLine(JsonConvert.SerializeObject(nextPage, Formatting.Indented));
        }
    }
}

File: official/docs/python/v9/pagination/get-next-page.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

# Get first page of results
shipments = client.shipment.all(
    page_size=5,
)

# Provide the previous results page to move onto the next page
next_page = client.shipments.get_next_page(shipments)

print(next_page)
